Calculator.py

```diff
@@ -17,3 +18,8 @@ def average(self, nums):
     nums_sum = self.sum(nums)
     result = nums_sum / l
     return result
+  def sine(self,nums):
+    result = []
+    for i in nums:
+        result.append(math.sin(i))
```

Comment:
use Python3 map to generate a list of sines from list of angles. Take a look at this PR https://github.com/itaditya/python-calculator/blob/master/Calculator.py#L22

You won't need lambda though

Reply:
Done